packages/suite/src/components/suite/modals/ReviewTransaction/components/Output.tsx

```diff
@@ -163,6 +160,15 @@ const Output = (props: Props) => {
                 plainValue: true,
             },
         ];
+    } else if (type === 'opreturn') {
+        outputLines = [
+            {
+                id: 'default',
```

Comment:
isn't id 'default' potentially dangerous? I am not very familar with send form. just asking if not better to make it unique

Reply:
10dd781bb